src/encoded/static/components/app.js

```diff
@@ -486,23 +489,25 @@ export default class App extends React.Component {
      */
     listActionsFor(category) {
         if (category === 'context') {
-            var context = this.props.context;
-            var name = this.currentAction();
-            var context_actions = [];
-            Array.prototype.push.apply(context_actions, context.actions || []);
+            var context         = this.props.context,
+                currentAction   = this.currentAction(),
+                contextActions = [];
+
+            Array.prototype.push.apply(contextActions, context.actions || []); // Add any from context itself
 
-            if (!name && context.default_page) {
+            // The below is deprecated ? TODO: Double check & delete.
```

Comment:
Are you leaving this if statement behind just in case you missed one behavior?

Reply:
I think the whole `category === 'context'` block can be:
```
if (category === 'context') {
            var context         = this.props.context,
                contextActions = [];

            Array.prototype.push.apply(contextActions, context.actions || []); // Add any from context itself
            return contextActions;
        }
```

Reply:
No idea what the `default_page` field is or is meant to be so left it in w/ the TODO comment.

Reply:
@alexkb0009 I don't think we use `default_page` (maybe a holdover from ENCODE?)

Up to you if you want to change; I think removing that code now is a good move for the sake of cleanliness